packages/engine-server/src/cache/dendronFileSystemCache.ts

```diff
@@ -31,7 +31,11 @@ export abstract class DendronFileSystemCache<T extends FileSystemCache, V>
    */
   private readFromFileSystem(): T {
     if (fs.existsSync(this._cachePath)) {
-      return fs.readJSONSync(this._cachePath) as T;
+      try {
+        return fs.readJSONSync(this._cachePath) as T;
+      } catch (_err: any) {
```

Comment:
What kind of errors did you see here? Can we log it?

Reply:
In case file gets corrupted or is an invalid json file. Sure I'll log it